faq/general-faq.md

```diff
@@ -0,0 +1,20 @@
+# FAQ
+Frequently Asked Questions
+
+### How do i give someone mod?
+Assign a role in phpmyadmin or use the GDPSFN panel, check out [This](phpmyadmin/create-roles.md) guide for a detailed explanation of permissions
+
+### Why does most of the panel not work?
+The panel is still under development, you can accomplish bascially everything the panel does in phpnyadmin.
+
+### Can i add 2.2? 
+Not without a mod, GDPSFN does not support 2.2 at this time
+
+### I can't complete 3 star and above levels that are shorter than 30 seconds
+You need to disable anticheat
```

Comment:
aha. good answer

Reply:
well i will link the remove anti-cheat page there

Reply:
Ok fixed